src/platform-includes/getting-started-primer/javascript.nextjs.mdx

```diff
@@ -13,6 +13,7 @@ Features:
 - [Breadcrumbs](/platforms/javascript/enriching-events/breadcrumbs/) created for outgoing HTTP request with XHR and Fetch, and console logs
 - [Release health](/product/releases/health/) for tracking crash-free users and sessions
 - Automatic [Performance Monitoring](/product/performance/) for both the client and server, from version `6.5.0`
+- Errors and Performance support for [Middleware](https://vercel.com/docs/concepts/functions/edge-middleware) and [Edge routes](https://vercel.com/docs/concepts/functions/edge-functions) in Vercels experimental edge runtime. Due to complexities with the experimental runtime, some features of errors like stacktraces may not not be as expected.
```

Comment:
```suggestion
- Errors and Performance support for [Middleware](https://vercel.com/docs/concepts/functions/edge-middleware) and [Edge routes](https://vercel.com/docs/concepts/functions/edge-functions) in Vercels experimental edge runtime. Because middleware and edge routes run in a sandboxed environment, stack traces may look funky and cannot be prettified with source maps.
```

I wouldn't shy away from being specific here.